### PR TITLE
Tab navigator

### PR DESCRIPTION
**Pull request by Kanishk-K**
Created a master Tab Navigator that allows switching between separate child Stack Navigators. Currently, only a Search Stack Navigator exists, however, more Stack Navigators can be created to navigate within their own screens as more screens are developed going forward.

### CODE REVIEW

File: react_frontend/components/TabNavigator.tsx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+                        </View>
+                )}} 
+            />
+        </Tab.Navigator>
+    );
+}
+const styles = StyleSheet.create({
+  navigator: {
+    position: 'absolute',
+    height: 90,
```
Comment (dnmalenke) [inline react_frontend/components/TabNavigator.tsx:98]:
```suggestion
    height: 69,
```

File: react_frontend/components/TabNavigator.tsx [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+
+function TabNavigator(){
+    const Tab = createBottomTabNavigator();
+    return (
+        <Tab.Navigator initialRouteName="Mainpage" screenOptions={{
+            tabBarShowLabel:false,
+            tabBarStyle:{
+                position: 'absolute',
+                height: 90,
```
Comment (dnmalenke) [inline react_frontend/components/TabNavigator.tsx:16]:
```suggestion
                height: 69,
```

**Review by dnmalenke — DISMISSED** [review]
image
It's beautiful but it do be lookin kinda **_THICC_**

**Comment by dnmalenke** [comment]
@Kanishk-K I would suggest finding how height of the top bar is determined and make it equal to that

**Review by e-d-v — DISMISSED** [review]

**Review by e-d-v — APPROVED** [review]

**Pull request merged**